_posts/2024-07-02-schematic-review-checklist.md

```diff
@@ -1,45 +1,45 @@
+Most of the schematic reviews I've attended are led by electrical engineers and are focused on the electrical components of the system. There are usually some questions about whether the proper diode was used or the capacitors were selected properly. Things that an electrical engineer should care about but typically fly over my head. As a firmware engineer, I've found it useful to think about what hardware features the code I'll end up writing will need. Starting from that perspective, I can leverage the schematic review as an opportunity to catch design issues while the fix is relatively cheap, before hardware has gone through various rounds of testing, which can be the difference between hitting a schedule or not.
 
-"Schematic review" here is mostly used as an easy blog post title, and a clear time for the firmware and electrical teams to interact. However, the important piece of this post is that you as a firmware engineer work with your electrical engineering counterpart to ensure you have the hardware you need to develop, debug, and deploy your code. And you identify those needs early on in the development cycle with the electrical team, when changes to PCBs are relatively cheap.
+"Schematic review" here is mostly used as an easy blog post title, and a clear time for the firmware and electrical teams to interact. However, the important piece of this post is that you as a firmware engineer work with your electrical engineering counterpart to ensure you have the hardware you need to develop, debug, and deploy your code. And you identify those needs early on in the development cycle with the electrical team when changes to PCBs are relatively cheap.
 
 Some pre-schematic review checklist items:
 
- - [ ] Get invited to the schematic review. Really, this is a callout to be working with your electrical engineer. If you've been working with your electrical engineer, providing guidance on what hardware you'll need, being invited to the schematic review should be a given.
- - [ ] Build your checklist up front. Use this blog post as a starting point, but make sure you customize the list for the given project.
+ - [ ] Get invited to the schematic review. Really, this is a callout to be working with your electrical engineer. If you've been working with your electrical engineer, providing guidance on what hardware you'll need, being invited to the schematic review should be a given.
```

Comment:
Minor formatting - what do you think of checking the boxes with `[x]`? Gives a nice feeling of completion to me for some reason. Here is a comparison:

![Screenshot 2024-07-16 at 4.09.10 PM.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/Fc91I7FkQVUXaB5l1YS5/7e6428a2-af10-4145-808c-45a8abb0bd7e.png)

Reply:
Side note, I wish just the checkbox got rendered and not the bullet, but I think this is just how it is if we use checkboxes